locks/unix/global_mutex.c

```diff
@@ -86,6 +86,10 @@ APR_DECLARE(apr_status_t) apr_global_mutex_child_init(
 {
     apr_status_t rv;
 
+    if (*mutex == NULL) {
+        return APR_ENOLOCK;
+    }
+
```

Comment:
This looks to me like an APR contract change, that we should docx the new return code and suggest to the author they can set up the appropriate retry schema, and put it back on mod_isapi.

Reply:
Specifically, for APR 2.0